src/pages/LendingTeam/TeamListing.vue

```diff
@@ -0,0 +1,312 @@
+<template>
```

Comment:
For tablet and mobile design has team listings displaying 5 teams (instead of 10) with button to expand to see more info about the team; all of this still needs to be implemented.